demo/admin/src/products/generator/ManufacturersGrid.cometGen.ts

```diff
@@ -1,14 +1,21 @@
 import { defineConfig } from "@comet/admin-generator";
 import { type GQLManufacturer } from "@src/graphql.generated";
 
+import { idFilterFilterOperators } from "../helpers/idFilterFilterOperators";
+
 export default defineConfig<GQLManufacturer>({
     type: "grid",
     gqlType: "Manufacturer",
     fragmentName: "ManufacturersGridFuture", // configurable as it must be unique across project
     queryParamsPrefix: "manufacturers",
     newEntryText: "Add Manufacturer",
     columns: [
-        { type: "text", name: "id", headerName: "ID" },
+        {
+            type: "text",
+            name: "id",
+            headerName: "ID",
+            filterOperators: idFilterFilterOperators,
```

Comment:
shall we probably provide a similar helper like we have for date/dateTime Columns: (see: https://github.com/vivid-planet/comet/blob/next/packages/admin/admin/src/dataGrid/gridColumnTypes.tsx#L46)

Maybe a `dataGridIdColumn`?

Reply:
duplicate 😅